ts/tx.ts

```diff
@@ -127,15 +127,17 @@ export class TxMassMigration implements SignableTx {
     }
 
     public message(): string {
-        const concated = concat([
-            this.TX_TYPE,
-            hexZeroPad(hexlify(this.fromIndex), stateIDLen),
-            hexZeroPad(this.amount.toHexString(), 32),
-            hexZeroPad(this.fee.toHexString(), 32),
-            hexZeroPad(hexlify(this.nonce), nonceLen),
-            hexZeroPad(hexlify(this.spokeID), spokeLen)
-        ]);
-        return hexlify(concated);
+        return solidityPack(
+            ["uint8", "uint32", "uint256", "uint256", "uint32", "uint32"],
```

Comment:
from index is `uint32` and to index is `uint256` not sure if it is a typo, just pointing out.

Reply:
Oh, that's not a toIndex, we don't have the toIndex now.

Reply:
Ah! Right, gotcha.